plugins/in_netif/in_netif.c

```diff
@@ -317,6 +293,31 @@ static int in_netif_init(struct flb_input_instance *in,
     return 0;
 }
 
+static struct flb_config_map config_map[] = {
+    {
+      FLB_CONFIG_MAP_STR, "interface", (char *)NULL,
+      0, FLB_TRUE, offsetof(struct flb_in_netif_config, interface),
+      "Set the interface"
```

Comment:
It may be good for user to add example. (e.g. eth0, enp0s3)

Reply:
I added it as an example. Setting as the default is pretty useless since most modern distributions use the new enp nomenclature.